packages/neo-one-smart-contract-compiler/src/test/setupContractTest.ts

```diff
@@ -1,4 +1,4 @@
-import { findAndCompileContract } from '@neo-one/smart-contract-compiler';
+import { findAndCompileContract } from '../';
```

Comment:
This makes a circular dependency on the `index.ts` file since we re-export this file there. In general, you should never import from `./` or `../`, `../../`, etc.

Change this to `../findAndCompileContract`. Similarly throughout the PR.